controllers/storagecluster/volumesnapshotterclasses.go

```diff
@@ -130,11 +130,13 @@ func (r *StorageClusterReconciler) createSnapshotClasses(vsccs []SnapshotClassCo
 
 // ensureCreated functions ensures that snpashotter classes are created
 func (obj *ocsSnapshotClass) ensureCreated(r *StorageClusterReconciler, instance *ocsv1.StorageCluster) (reconcile.Result, error) {
-	vsccs := newSnapshotClassConfigurations(instance)
+	if !IsOCSConsumerMode(instance) {
```

Comment:
instead of this change how about https://github.com/red-hat-storage/ocs-operator/blob/main/controllers/storagecluster/reconcile.go#L405 add ocsSnapshotClass only if its not in consumerMode?

Reply:
changed